packages/components/tabs/src/tabs.ts

```diff
@@ -204,7 +210,12 @@ export default defineComponent({
       (modelValue) => setCurrentName(modelValue)
     )
 
-    watch(currentName, async () => {
+    watch(
+      () => props.scrollPosition,
+      (scrollValue) => (scrollPosition.value = scrollValue)
```

Comment:
**Question:** why can't we just use `props.scrollValue` directly since it is already reactive. If we introduce a dedicated ref for this value I don't see the point. If this is really needed, don't you think `computed` would be an better idea?

```ts
const scrollPosition = computed(() => props.scrollValue)
```

Reply:
My bad, fixed